_rules/zoom-text-no-overflow-clipping-59br37.md

```diff
@@ -0,0 +1,247 @@
+---
+id: 59br37
+name: Zoomed text node is not clipped with CSS overflow
+rule_type: atomic
+description: |
+  This rule checks that text nodes are not unintentionally clipped by overflow, when a page is zoomed to 200% on 1280 by 1024 display;
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:1.4.4: # Resize Text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: urther testing needed
+    inapplicable: urther testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+  assets:
+    - The raven, Edgar Allan Poe
+---
+
+## Applicability
+
+The rule applies to any [text node][] for which all of the following is true when in a [display size][] of 640 by 512:
+
+- The [text node][] is [visible][]; and
+- The [text node][] has an HTML element as a [parent][] in the [flat tree][]; and
+- The [text node][] has an [ancestor][] in the [flat tree][] with the a [computed][] [overflow-x][overflow] or [overflow-y][overflow] of `none` or `clip`; and
+- The [text node][] does not have an [ancestor][] in the [flat tree][] with an `aria-hidden` attribute set to `true`
+
+**note**: A [display size][] of 640 by 512 is equivalent to a [display size][] of 1280 by 1024 zoomed 200%.
+
+## Expectation
+
+Each test target is not [clipped by overflow][clipped] of an [ancestor][] in the [flat tree][] when in a [display size][] of 640 by 512, except if the [clipping][clipped] [ancestor][] has one of the following:
+
+- **text-overflow**: A [computed][] [white-space][] of `nowrap`, and a [computed][] [text-overflow][] that is not `clip`; or
+
+- **line wrapping**: A [computed][] [line-height][] equal to or greater than its
+  height of its [bounding box][], or in case of a [computed][] [overflow][] of `clip`, its [content box][].
+
+## Assumptions
+
+If any of the following assumptions is true, failing this rule may not result in a failure of [success criterion 1.4.4 Resize text](https://www.w3.org/TR/WCAG21/#resize-text):
+
+- There is no other mechanism for resizing text available on the page, that can be used to resize text to 200% without loss of information or functionality. This includes font resizing in the browser, or a javascript mechanism of resizing in the page.
```

Comment:
> This includes font resizing in the browser [...]

At first, I was a little hesitant about this bit. On second thought, I do like what this implies though; if a user cannot zoom to 200% without loss of information, it shouldn't really matter that they might instead be able to bump the base font size to 200% and not lose information.